libsolidity/formal/SMTChecker.cpp

```diff
@@ -609,6 +610,12 @@ void SMTChecker::endVisit(FunctionCall const& _funCall)
 		popCallStack();
 		break;
 	case FunctionType::Kind::External:
+	case FunctionType::Kind::DelegateCall:
+	case FunctionType::Kind::BareCall:
+	case FunctionType::Kind::BareCallCode:
+	case FunctionType::Kind::BareDelegateCall:
+	case FunctionType::Kind::BareStaticCall:
+	case FunctionType::Kind::Creation:
```

Comment:
`Creation` is not an address member, but since it would be the last of the external calls I added it here. I can also create a separate PR for it if needed.